### PR TITLE
fixed log not generating the log directory

### PR DESCRIPTION
**Pull request by ThinkRedstone**

### CODE REVIEW

File: server_modules/log.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ var rotate = require('rotating-file-stream');
 
 const LOG_PATH = path.resolve(__dirname, '..', 'log', 'log.log');
 
+if (!fs.existsSync(path.dirname(LOG_PATH))) {
+    fs.mkdirSync(path.dirname(LOG_PATH))
```
Comment (idanstark42) [inline server_modules/log.js]: This line creates a folder called `.../log/log.log`

Comment (ThinkRedstone) [reply server_modules/log.js]: This creates a folder named log in __dirname/.. on my computer

Comment (idanstark42) [reply server_modules/log.js]: And inside of it?
https://nodejs.org/api/fs.html#fs_fs_mkdir_path_mode_callback

Comment (ThinkRedstone) [reply server_modules/log.js]: a file named log.log

Comment (idanstark42) [reply server_modules/log.js]: Oh. Sorry, didn't notice that

**Pull request merged**